<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommParamSetDlg</class>
 <widget class="BaseDlg" name="CommParamSetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">CommParamSetDlg</string>
  </property>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>817</x>
     <y>810</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>947</x>
     <y>810</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="seriial_port_set_radiobtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>448</y>
     <width>445</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>串口设置</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="tcp_set_radiobtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>448</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>网络设置</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>498</y>
     <width>445</width>
     <height>287</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">all</string>
   </property>
   <widget class="QComboBox" name="parity_check_cbox">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>48</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>8</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>182</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>95</y>
      <width>201</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QComboBox" name="seriial_port_cbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>48</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QComboBox" name="data_bit_cbox">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>135</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QComboBox" name="stop_bit_cbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>135</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>95</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baud_tate_cbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>222</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>8</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>奇偶校验</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>498</y>
     <width>220</width>
     <height>287</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="borders" stdset="0">
    <string>all</string>
   </property>
   <widget class="QLineEdit" name="service_port_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>135</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>8</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="service_ip_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>48</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>95</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>138</y>
     <width>325</width>
     <height>296</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="borders" stdset="0">
    <string>all</string>
   </property>
   <widget class="QLineEdit" name="wait_time_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_trans_times_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>226</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>188</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>最大传输次数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>接收等待时间(S)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comm_protocol_cbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>HL7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ASTM</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>138</y>
     <width>675</width>
     <height>296</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="borders" stdset="0">
    <string>all</string>
   </property>
   <widget class="QCheckBox" name="receive_sample_info_ckbox">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>20</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>接收样本信息</string>
    </property>
    <property name="checks" stdset="0">
     <string>push</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="receive_sample_num_ckbox">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>108</y>
      <width>261</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>接收样本号</string>
    </property>
    <property name="checks" stdset="0">
     <string>push</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="auto_upload_results_gbox">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>75</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>自动上传结果</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checks" stdset="0">
     <string notr="true">push</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>37</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>检测完成上传</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>77</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>审核完成上传</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="auto_upload_sample_num_ckbox">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>148</y>
      <width>261</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>上传样本号</string>
    </property>
    <property name="checks" stdset="0">
     <string>push</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="receive_recheck_sample_ckbox">
    <property name="geometry">
     <rect>
      <x>357</x>
      <y>20</y>
      <width>301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>接收复查样本</string>
    </property>
    <property name="checks" stdset="0">
     <string>push</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>65</y>
      <width>301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>条码模式</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="comm_params_id">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>281</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>通讯ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="comm_params_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>通讯名称</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="comm_link_ckbox">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>88</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>通讯连接</string>
   </property>
   <property name="checks" stdset="0">
    <string notr="true">push</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>88</y>
     <width>355</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>通讯协议</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>498</y>
     <width>325</width>
     <height>287</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="borders" stdset="0">
    <string>all</string>
   </property>
   <widget class="QLineEdit" name="test_rlt_path_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>135</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>8</y>
      <width>331</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>样本请求路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sample_req_path_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>48</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>95</y>
      <width>331</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>结果路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sample_assay_path_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>222</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>182</y>
      <width>331</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>样本项目路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse1_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>45</y>
      <width>96</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
    <property name="buttons" stdset="0">
     <string>btn_dlg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse2_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>132</y>
      <width>96</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
    <property name="buttons" stdset="0">
     <string>btn_dlg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse3_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>219</y>
      <width>96</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
    <property name="buttons" stdset="0">
     <string>btn_dlg</string>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="file_path_radiobtn">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>448</y>
     <width>321</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>CommParamSetDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>977</x>
     <y>693</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
